src/components/shared/Button/ThreeDotsButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+import classnames from 'classnames';
+
+import Icon from '~shared/Icon';
+
+import Button, { Props as DefaultButtonProps } from './Button';
+import styles from './ThreeDotsButton.css';
+
+const displayName = 'Button.ThreeDotsButton';
+
+interface Props extends DefaultButtonProps {
+  activeStyles?: string;
+  isOpen: boolean;
+  title: MessageDescriptor;
+  innerRef?: (ref: HTMLInputElement | null) => void;
+}
+
+const ThreeDotsButton = ({
```

Comment:
Isn't @arrenv  handling this in #143 ?

Reply:
The ThreeDotsButton existed here before #143 was created. I already made sure the changes are identical.